packages/web/src/migrations/update-11-5-0/update-babel-config.ts

```diff
@@ -0,0 +1,61 @@
+import { chain, Rule, Tree } from '@angular-devkit/schematics';
+import { getFullProjectGraphFromHost } from '@nrwl/workspace/src/utils/ast-utils';
+import { formatFiles, updateJsonInTree } from '@nrwl/workspace';
+
+export default function update(): Rule {
+  return (host: Tree) => {
+    const updates = [];
+    const projectGraph = getFullProjectGraphFromHost(host);
+
+    if (host.exists('babel.config.json')) {
+      updates.push(
+        updateJsonInTree('babel.config.json', (json) => {
+          if (Array.isArray(json.presets)) {
+            json.presets = json.presets.filter((x) => x !== '@nrwl/web/babel');
+          }
+          return json;
+        })
+      );
+    }
+
+    Object.keys(projectGraph.nodes).forEach((name) => {
```

Comment:
If we write this as a `@nrwl/devkit` generator then you can use `getProjects()` to get a list of the projects in the workspace. which is cheaper than getting the project graph because it won't calculate dependencies.

Reply:
Yup, all migrations scripts have to be written using Nx Devkit.